src/main/resources/db/migration/tenants/V1__init_schema.sql

```diff
@@ -1,7 +1,8 @@
 CREATE TABLE "author"
 (
     "id" SERIAL PRIMARY KEY NOT NULL,
-    "name" TEXT
+    "name" TEXT,
+    CONSTRAINT fk_author_globally_unique FOREIGN KEY (id) REFERENCES "default".users(id)
```

Comment:
This also wrong author.id is not same as user.id
the issue was in note table which contains foreign keys to all schema

Reply:
This query:

```sql
SELECT conrelid::regclass AS table_from
     , conname
     , pg_get_constraintdef(oid)
FROM   pg_constraint
WHERE  contype IN ('f', 'p ')
AND conrelid::regclass::text  like '%.note' and conname like '%fk%'
ORDER  BY conrelid::regclass::text, contype DESC;
```

Gives the following FK's in this branch:
![image](https://github.com/jeebendu/schema-based-multi-tenancy/assets/1144011/ee6e4a73-9a11-47d6-b94e-519a83d70b1b)


Is that not the desired relationship between note/author?

Reply:
i am using the heidisql for sql client , might be the issue from here
in PG admin it is showing correctly

which SQL client you are using ?

Reply:
https://github.com/HeidiSQL/HeidiSQL/issues/1956
created a ticket in HeidiSQL github issues